views/js/qtiXmlRenderer/renderers/RendererPerInteractionRP.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ *
+ */
+define([
+    'taoQtiItem/qtiRunner/core/Renderer',
+    'taoQtiItem/qtiXmlRenderer/renderers/config'
+],
+    function (Renderer, config) {
+        'use strict';
+
+        return Renderer.build(
+            {
+                ...config.locations,
```

Comment:
Can't be build, we don't support this.
`npx grunt taoqtiitembundle`
![image](https://user-images.githubusercontent.com/25976342/93848478-50e84900-fcb2-11ea-8d16-b7a51754833c.png)